src/main.rs

```diff
@@ -178,3 +179,35 @@ async fn main() -> Result<(), Error> {
 
     Ok(())
 }
+
+fn make_message(pull_request: GithubPullRequest, show_pr_age: bool) -> String {
+    let mut message = "".to_string();
+
+    write!(message, "<{}|{}#{}> - {}",
+        pull_request.html_url().replace("https://", ""),
+        pull_request.head().repo().name(),
+        pull_request.number(),
+        pull_request.title()).unwrap();
+
+    if show_pr_age {
+        let age = get_age(Utc::now().with_timezone(&FixedOffset::east_opt(0).unwrap()), *pull_request.created_at()).to_string();
+        write!(message, " - (_{}_)", age).unwrap();
+    }
+
+    write!(message, "\n").unwrap();
+    return message;
+}
+/// Returns a string representing the time between two dates in the format of
+/// "opened {value} {unit} ago".
+/// 
+/// Example: "opened 1 day ago"
+fn get_age(d1: DateTime<FixedOffset>, d2: DateTime<FixedOffset>) -> String {
+    let difference = d1.signed_duration_since(d2);
+    let mut output = "".to_string();
+    write!(output, "opened {} ago", match difference.num_days() {
+        0 => "less than a day".to_string(),
+        1 => "1 day".to_string(),
+        i => format!("{} days", i)
+    }).unwrap();
+    return output;
```

Comment:
If you don't need to concatenate strings `format!()` might be a better option here, it does the same thing as `write!()` but returns a string instead of adding it to an existing one!

```suggestion
    format!("opened {} ago", match difference.num_days() {
        0 => "less than a day".to_string(),
        1 => "1 day".to_string(),
        i => format!("{} days", i)
    }).unwrap()
```

Also the same thing with unecessary returns!